api4/channel_test.go

```diff
@@ -291,7 +291,10 @@ func TestPatchChannel(t *testing.T) {
 	_, resp = th.SystemAdminClient.PatchChannel(th.BasicChannel.Id, patch)
 	CheckNoError(t, resp)
 
-	_, resp = th.SystemAdminClient.PatchChannel(th.BasicPrivateChannel.Id, patch)
+	Client.Logout()
+	Client.Login(th.BasicUser.Username, th.BasicUser.Password)
+
+	_, resp = th.Client.PatchChannel(th.BasicPrivateChannel.Id, patch)
```

Comment:
Should we add test cases to both patch and update channel that verify that the membership check is working?

Reply:
Yes good point, will add them.

Reply:
Done.